src/nvim/screen.c

```diff
@@ -1570,6 +1570,17 @@ static void win_update(win_T *wp)
     got_int = save_got_int;
 }
 
+///
+/// Returns maximum length of the signcolumn
+///
+/// @param wp window we want signcolumn width from
+///
+/// @return max width of signcolumn (cell unit)
+int win_get_colwidth(win_T *wp)
+{
+  // 2 is the default value
+  return 2;
```

Comment:
I'm confused by the function name, the docstring, and the actual usage. Is it a general function for the leftmost column width, or only the sign column? Is it intended to always return the maximum?

Is the `wp` parameter a signal that it's going to be made window-specific in a future change?